City-of-Helsinki/helsinki-paatokset
# PP-297: Changes to voting results

Ticket: https://helsinkisolutionoffice.atlassian.net/browse/PP-297

Changed text color to white if background is red, fixed column widths, disabled accordion button if number of votes is 0, semantic fixes to headers Äänestys [index] should be h3 and the ones below h4

**How to test**

Checkout branch and go to i.ex. https://helsinki-paatokset.docker.so/fi/asia/hel-2021-002856/3e4074e1-8b43-4abf-a61a-d4d672b7c5c3 (requires that all decisions are migrated)

Changes mentioned above should now be in the voting result accordion.
